src/core/adminVault.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/utils/Pausable.sol";
+
+contract superAdminVault is AccessControl, ReentrancyGuard, Pausable {
```

Comment:
file name is `adminVault.sol` but the contract is `superAdminVault.sol`, should changes the contract name or the file name for consistency.